threedi_modelchecker/tests/conftest.py

```diff
@@ -39,7 +39,7 @@ def session(threedi_db):
 
     :return: sqlalchemy.orm.session.Session
     """
-    s = threedi_db.get_session()
+    s = threedi_db.get_session(future=True)
```

Comment:
Maybe good to add a corresponding session.close() ?

Reply:
The session seems to be used for the duration of the test run for all the tests, and then is closed by the garbage collector anyway. Adding `s.close()` leads to no performance change in pytest, either.